content/docs/support-and-ops/ops-recipes/configure-etcdbackup-exclusions.md

```diff
@@ -0,0 +1,58 @@
+---
+title: "Configure ETCDbackup Exclusions"
+owner:
+- https://github.com/orgs/giantswarm/teams/team-planeteers
+component:
+  - etcd-backup-operator
+description: "How to configure ETCDbackup custom resource to exclude specific clusters from backups"
+classification: public
+---
+
+## Overview
+
+The ETCDbackup custom resource can be configured to exclude specific clusters from being backed up. This is particularly useful when you need to downscale or decommission clusters, as it prevents unnecessary backup operations.
```

Comment:
```suggestion
The ETCDBackup custom resource can be configured to exclude specific clusters from being backed up. This is particularly useful when you need to downscale or decommission clusters, as it prevents unnecessary backup operations.

__Note__: Scaling masters to zero will destroy the cluster, which cannot be restored anymore. It will need a backup to be restored.
```